spec/ysh-builtin-eval.test.sh

```diff
@@ -109,7 +109,7 @@ foo bar baz
 #### eval lines with argv bindings
 proc my-split (;;; block) {
   while read --raw-line {
-    var cols = _reply => split()
+    var cols = split(_reply)
```

Comment:
oh yeah I forgot we have both a free function and a method

now that we are getting rid of `=>` in favor of `.`, there's a one char difference

```
split(s)
s.split()
```

We can decide on whether we need both separately

I suppose it is inconsistent if we have `split()` and not others

---

I think `shSplit()` should possibly be a free function ...   it was related to glob(), because the traditional shell word eval algorithm  uses IFS splitting and globbing

Reply:
> I think `shSplit()` should possibly be a free function ... it was related to glob(), because the traditional shell word eval algorithm uses IFS splitting and globbing

Yeah, I agree that those are fine as free functions for that reason.

However, `split` really should only be a method. Let's leave it in for now though as otherwise there is no way to get a python style `string.split()` (split on all runs of whitespace.)